ios/PurchasesHybridCommon/PurchasesHybridCommonTests/PurchasesHybridCommonTests.swift

```diff
@@ -361,5 +361,20 @@ class PurchasesHybridCommonTests: QuickSpec {
                 }
             }
         }
+
+        context("setLogHandler") {
+            let expectedMessage = "a message"
+            for level in LogLevel.levels {
+                it("\(level) logs work") {
+                    let originalHandler = Purchases.verboseLogHandler
```

Comment:
Ok so I noticed that doing this:

```
let originalHandler = Purchases.logHandler
Purchases.logHandler = originalHandler
```

Makes the code loop forever until it crashes. The getter creates a new function (function `a`) that calls the `verboseLogHandler`. Then we set `logHandler` to that new function `a`. But the setter of `logHandler` actually sets `verboseLogHandler` to a new function that calls function `a`. The problem is that `function a` was set to call `verboseLogHandler`. So when calling `verboseLogHandler` we will call `function a`, which will call `verboseLogHandler` again, and a loop is created

Reply:
So I decided to just set the `verboseLogHandler` instead.

Reply:
Oh yeah that makes sense :/ that's a consequence of the fact that `logHandler` is derived from `verboseLogHandler`.

This implementation makes sense though 👍🏻 

To clean it up I would move it out of the test though:
```swift
context("setLogHandler") {
  var originalHandler: VerboseLogHandler!
  beforeEach {
    originalHandler = Purchases.verboseLogHandler
  }

  afterEach {
    Purchases.verboseLogHandler = originalHandler
  }

  it("...") {}
}
```